config/configuration.json

```diff
@@ -131,6 +131,28 @@
       "difficulty": "MEDIUM"
     }
   },
+  "MixedCaseNameCheck": {
+    "check_name.countries":["AIA", "ATG", "AUS", "BHS", "BRB", "BLZ", "BMU", "BWA", "VGB",
```

Comment:
Please use the standardized, "countries.whitelist". Then the country list will be handled automatically by the framework and you don't have to add all the extra code.

Reply:
I think this needs to be more clearly defined in the documentation. Why wouldn't you check the name tag and name:[language] tags regardless of whether the ISOCountryTag exists or not.

Reply:
The idea is that you only want to check names that are in a language that the check is configured to handle. By default the check is configured for english. So, you only want to check the `name` tag when the country is english speaking, but `name:en` can be checked in any country because it will always be in english. I will see about clarifying that in the docs.

Reply:
I tried to make this a little clearer in https://github.com/osmlab/atlas-checks/pull/66/commits/6d34f4dd832360d43ae113cb12bab1c0c2d9e518. Let me know if you still think it needs more clarification.